posts/2016-12-02-serverless-v1.3.0.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Function metrics, AWS credential setup and Lambda versioning - Serverless Framework v1.3
+description: Function metrics, easier AWS credential setup and Lambda versioning available in Serverless v1.3
+date: 2016-12-02
+layout: Post
+---
+
+We're proud to release the Serverless Framework v1.3.0 today.
+
+## Highlights of 1.3.0
+
+Let's take a look at the highlights of this release. You can find a list with all the changes in the [release changelog](https://github.com/serverless/serverless/releases/tag/v1.3.0).
+
+### Function Metrics
+
+Ever wondered how your functions are performing? The new `serverless metrics` lets you see all the function metrics such as invocations, throttles, errors and duration at a glance.
+
+Just type in
+
+```bash
+serverless metrics --function myFunction
+```
+
+to see the metrics of the last 24h for your function `myFunction`.
+
+You can even specify the timeframe you want to see the metrics with the `startTime` and `endTime` options.
+
+```bash
+serverless metrics --function myFunction --startTime 2016-11-28 --endTime 2016-12-02
+```
+
+will show you all the metrics between November, 28th and December, 2nd.
+
+You can read more about the new metrics functionality in [the documentation](https://serverless.com/framework/docs/providers/aws/cli-reference/metrics/).
+
+### AWS Credential Setup
+
+Setting up credentials should be streamlined in the overall Serverless experience. v1.3 adds a new command called `serverless config credentials` which helps you setup your AWS crendentials on your local machine.
+
+This example will setup the `default` AWS profile with the provided key and secret:
+
+```bash
+serverless config credentials --provider aws --key someKey --secret someSecret
+```
+
+No need to configure the `credentials` file through the `aws-cli` anymore.
+
+You can read more about the new config functionality in [the documentation](https://serverless.com/framework/docs/providers/aws/cli-reference/config/).
+
+### Lambda versioning
+
+Keeping a history of all your deployed Lambda functions is an important feature. Serverless now supports automatic Lambda versioning with the help of the build in Lambda versions AWS provides.
```

Comment:
Without @mthenw just explaining it to me 5 min ago I wouldn't know why this is important. Can we change this here to explaining that any log you print now contains the function version and therefor it makes it easier to identify bugs.